.astylerc

```diff
@@ -4,10 +4,9 @@
 --indent-labels
 --indent-col1-comments
 --min-conditional-indent=0
---break-blocks
```

Comment:
Actually I did not complain for quite a long time about that. :) Mostly because most new commits contain this extra line (possibly because of this astyle parameter). IMO the option can stay, I don't mind.